gatsby-node.js

```diff
@@ -3,41 +3,3 @@
  *
  * See: https://www.gatsbyjs.com/docs/node-apis/
  */
-
```

Comment:
removed all code from gatsby-node, because the site no longer needs to build dynamic detail pages for projects